CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ else()
     find_package(Lua)
 endif()
 
-find_package(Boost 1.53.0 COMPONENTS system REQUIRED)
+find_package(Boost 1.55.0 COMPONENTS system REQUIRED)
```

Comment:
Travis only whitelists 1.55 and the latest from Ubuntu Precise, which is too old: https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise#L3854-L3934

I forgot to link this on the PR message :P